lib/evision_mat.ex

```diff
@@ -427,7 +427,7 @@ defmodule Evision.Mat do
              encoded <- Evision.imencode(".png", mat),
              true <- is_binary(encoded) do
           image = Kino.Image.new(encoded, :png)
-          tabs = Kino.Layout.tabs([{"Raw", raw}, {"Image", image}, {"Numerical", numerical}])
+          tabs = Kino.Layout.tabs([{"Image", image}, {"Raw", raw}{"Numerical", numerical}])
```

Comment:
```suggestion
          tabs = Kino.Layout.tabs([{"Image", image}, {"Raw", raw}, {"Numerical", numerical}])
```
Hi @tusqasi! Thank you for the contribution! I agree it makes sense to show an image by default where is possible. 

Adding this missing comma and we'll be good to go!